build.gradle.kts

```diff
@@ -38,12 +38,12 @@ kotlin {
             compilations.all {
                 kotlinOptions {
                     jvmTarget = "1.8"
-                    apiVersion = "1.7"
-                    languageVersion = "1.7"
```

Comment:
Since we're not using newer features, can we leave these as is?

Reply:
Yes. However, if we leave the `languageVersion = "1.7"`, the following error occurs, because the `commonMain` source set's language version is automatically determined as `1.9`:

```
Caused by: org.gradle.api.InvalidUserDataException: Inconsistent settings for Kotlin source sets: 'jvmMain' depends on 'commonMain'
'jvmMain': language version is KOTLIN_1_7
'commonMain': language version is KOTLIN_1_9
The language version of the dependent source set must be greater than or equal to that of its dependency.
```

So we would have to add this block to the `commonMain` configuration:
```
languageSettings {
    languageVersion = "1.7"
}
```
Is that ok with you?

Reply:
bumping only the languageVersion seems fine!